test/kustomization/expected.yaml

```diff
@@ -5650,6 +5652,45 @@ data:
   otel-agent-config.yaml: |
     receivers:
       opencensus:
+        endpoint: 0.0.0.0:55678
+    exporters:
+      opencensus:
+        endpoint: otel-collector.config-management-monitoring:55678
+        tls:
+          insecure: true
```

Comment:
this probably isn't new, but do we care that our metrics are being sent unencrypted between the agents and otel-collector?

/cc @janetkuo

Reply:
Agreed that we shouldn't use insecure, but given that it's unrelated to this PR, let's track that fix separately.

Reply:
For this topic I created b/402998344 to followup.